bench/workgen/runner/example_dynamic_tables.py

```diff
@@ -106,13 +117,46 @@ def create(session, workload, table_config):
 workload_thread = ThreadWithReturnValue(target=workload.run, args=([connection]))
 workload_thread.start()
 
-# Create tables while the workload is running.
+# Create and drop tables while the workload is running.
+tables_to_delete = []
 while workload_thread.is_alive():
     create(session, workload, table_config)
+
+    # Select a random table to drop.
+    if len(tables):
+        idx = random.randint(0, len(tables) - 1)
+        uri = tables[idx]
+        # Mark it for deletion.
+        drop(workload, uri)
+
+    # Trigger the Workgen garbage collection.
+    tables_to_delete += workload.garbage_collection()
+    deleted_tables = []
+    for t in tables_to_delete:
+        try:
+            session.drop(t)
+            # If the WiredTiger call is successful, this table has been removed safely from all
+            # layers.
+            deleted_tables.append(t)
```

Comment:
Here, WT could delete the table, hence we add it to `deleted_tables`. Any table that is inside the `deleted_tables` variable has been deleted (in order) from Python, Workgen and WiredTiger.